komga-webui/src/views/Home.vue

```diff
@@ -163,14 +185,37 @@ export default Vue.extend({
     isAdmin (): boolean {
       return this.$store.getters.meAdmin
     },
+    themeItems (): any[] {
+      return Object.values(Theme).map(theme => (
+        {
+          text: theme,
+          value: theme,
+        }
+      ))
+    },
```

Comment:
this doesn't need to be computed, it can be in data. `any[]` is also not recommended